docs/src/app/components/pages/components/Slider/ExampleValue.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Slider from 'material-ui/Slider';
+
+export default class SliderExampleValue extends React.Component {
+
+  state = {
+    firstSlider: 0.5,
+    secondSlider: 50,
+  }
+
+  handleSlider(key, value) {
+    this.setState({[key]: value});
+  }
+
+  render() {
+    return (
+      <div>
+        <Slider
+          defaultValue={0.5}
+          value={this.state.firstSlider}
+          onChange={(event, value) => this.handleSlider('firstSlider', value)}
```

Comment:
Well, the idea is to prevent the creating of a function in the render method and, in some aspect, increase readability / reduce the cost of changes. 
Here is an [example](https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/Dialog/ExampleSimple.js#L14.
).
```jsx
handleOpen = () => {
  this.setState({open: true});
};

...

<RaisedButton label="Dialog" onTouchTap={this.handleOpen} />
```